src/main/resources/com/itiviti/parser/ProjectFileParser.csproj

```diff
@@ -5,6 +5,7 @@
     <TargetFrameworks Condition="'$(FrameworkVersion)'==''">netcoreapp3.1;net5.0</TargetFrameworks>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
+    <WarningsNotAsErrors>NU1901;NU1902;NU1903;NU1904</WarningsNotAsErrors>
```

Comment:
We should rather fix the dependencies to not make use of the vulnerable dependencies actually.